### PR TITLE
Add Emc Oem Catalog, Poller and Compose graph

### PR DESCRIPTION
**Pull request by jlongever**
- Redfish System and Chassis pollers
- Platform-A Catalog and Compose

@RackHD/corecommitters @zyoung51 @geoff-reid @uppalk1  @tannoa2 @keedya 

Depends on https://github.com/RackHD/on-tasks/pull/145

### CODE REVIEW

File: spec/lib/graphs/create-redfish-chassis-poller-graph-spec.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+describe(require('path').basename(__filename), function () {
+    var base = require('./base-graph-spec');
+
+    base.before(function (context) {
+            context.taskdefinition = helper.require('/lib/graphs/create-redfish-chassis-poller-graph.js');
+        });
+
+    describe('graph', function () {
+            base.examples();
+        });
```
Comment (yyscamper) [inline spec/lib/graphs/create-redfish-chassis-poller-graph-spec.js]: a minor suggestion, indent error, the same for other files.

**Comment by yyscamper** [comment]
:+1:

**Comment by zyoung51** [comment]
+1

**Comment by tannoa2** [comment]
+1

**Comment by jlongever** [comment]
test this please

**Pull request merged**